### PR TITLE
migrate vit_small_ds.py to keras 3

### PR DESCRIPTION
**Pull request by lkarthee**

### CODE REVIEW

File: examples/vision/vit_small_ds.py [hunk 1]
```diff
@@ -49,11 +49,12 @@
 
 import math
 import numpy as np
+import keras
```
Comment (lkarthee) [inline examples/vision/vit_small_ds.py:52]: I have only updated py file. I will update other files after review.

**Review by fchollet — COMMENTED** [review]
LGTM, thank you. Does it run fine with all backends?

**Comment by lkarthee** [comment]
I ran on colab with default backend - it took long time to finish. Will CI check for all backends ?

**Comment by fchollet** [comment]
> Will CI check for all backends ?

No, it will not. Can you check by hand? No need to leave it running to completion, you can interrupt after `fit()` starts progressing.

**Pull request closed without merging**